invenio_rdm_records/theme/templates/invenio_rdm_records/record_landing_page.html

```diff
@@ -28,23 +28,21 @@
         <div class="ui grid">
           <div class="two column row">
             <div class="left floated left aligned column">
-              <span class="ui"
-                title="Publication date">{{ metadata.publication_date|to_date|format_date(format='long') }}</span>
+              <span class="ui" title="Publication date">{{ record.ui.publication_date_l10n }}</span>
```

Comment:
Minor: This should be in long format, so we need both a long and a short formatted publication date.